### PR TITLE
[KAMELEOON] added type property for every event

### PR DESCRIPTION
**Pull request by akalyuzhnyi**
The [Kameleoon](https://www.kameleoon.com/) cloud destination.

Testing
Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing.

added type property for every event

### CODE REVIEW

**Comment by joe-ayoub-segment** [comment]
Hi @akalyuzhnyi this PR doesn't look right. Didn't we just deploy some new Actions for the Kameleoon Destination via this PR? https://github.com/segmentio/action-destinations/pull/1974

Looks like this PR is doing something similiar?

**Comment by akalyuzhnyi** [comment]
Hello @joe-ayoub-segment 

Sorry there was wrong MR, now everything is ok

I forgot to add types for the event, could you review please?

File: packages/destination-actions/src/destinations/kameleoon/group/index.ts [hunk 1]
```diff
@@ -22,6 +22,15 @@ const action: ActionDefinition<Settings, Payload> = {
       label: 'User ID',
       default: { '@path': '$.userId' }
     },
+    type: {
+      label: 'Type',
+      type: 'string',
+      required: true,
```
Comment (joe-ayoub-segment) [inline packages/destination-actions/src/destinations/kameleoon/group/index.ts:28]: Hi @akalyuzhnyi - adding required:true means that all current/existing instances of the Integration will start to fail. 
So any customer who is already using the Integration will have their data collection stopped. 
Is this OK?

**Comment by joe-ayoub-segment** [comment]
Hi @akalyuzhnyi the change mostly looks good. However you need to be super careful about adding a new 'required' field. Please see comment above. 

If you have customers already using this Integration, I would make the type field optional.

**Comment by akalyuzhnyi** [comment]
yes, same type like before for logEvent :)

**Comment by joe-ayoub-segment** [comment]
> yes, same type like before for logEvent :)

hi @akalyuzhnyi - if I deploy the code as it is now it will break any data collection for customers who are already using it. 
Is this OK for you?

**Review by joe-ayoub-segment — APPROVED** [review]

**Comment by joe-ayoub-segment** [comment]
note: PR deployed on Tuesday this week

**Comment by akalyuzhnyi** [comment]
> note: PR deployed on Tuesday this week

Hello! everything works fine, thank you!

**Pull request merged**